### PR TITLE
2 Animation Scripts

### PR DESCRIPTION
**Pull request by kolibril13**
I want to contribute two scripts for the gallery.
This time in the correct folder

### CODE REVIEW

**Comment by kolibril13** [comment]
RotationAroundAxes

**Comment by kolibril13** [comment]
hmm, the " Create own VMobject " seems to be ok, but " RotationAroundAxes.py" and "ParamFunc" have somehow failed the tests. So I deleted "RotationAroundAxes.py". 
Maybe you can bring ParamFunc to work nevertheless.
Do you have an idea what the problem might be?

File: src/examples/02_animations/custom_VMobject.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+from manimlib.imports import *
+
+
+class MyDotGrid(VMobject):
+    def __init__(self):
+        VMobject.__init__(self)
+        dot1 = Dot(fill_color=GREEN).shift(LEFT)
```
Comment (soerface) [inline src/examples/02_animations/custom_VMobject.py:7]: The colors are not defined (at least in manimlib 0.1.6, but manim-gallery depends on this version)

File: src/examples/02_animations/parametric_function.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+from manimlib.imports import *
+
+
+def i_need_a_name(t):
```
Comment (soerface) [inline src/examples/02_animations/parametric_function.py:4]: What exactly does this function, is there a mathematical term for what happens here?

**Review by soerface — CHANGES_REQUESTED** [review]
Left some comments here. By the way, you don't need to close the pull request like you did with the last one. Just add more commits as you would usually do, they will be added to the PR automatically! :)

**Comment by kolibril13** [comment]
Ok, 
then I know for next time. :-)
I am not home this weekend, but next week I will make the changes.

File: src/examples/02_animations/custom_VMobject.py (hunk 1 above)
Comment (soerface) [reply src/examples/02_animations/custom_VMobject.py:7]: @kolibril13 Hrm - I just saw that in the other pull request, those colors are also used. PyCharm complains about them not being defined, but actually they are. Those damn `*` imports…

Comment (kolibril13) [reply src/examples/02_animations/custom_VMobject.py:7]: I know, this is a little annoying..

**Pull request closed without merging**